requirements.txt

```diff
@@ -58,6 +58,6 @@ six==1.10.0
 git+https://github.com/bartTC/django-attachments.git
 git+https://github.com/CSnap/django-likes.git
 git+https://github.com/CSnap/django_teams.git
+git+https://github.com/RuofanLiu/SGIS-backend.git
```

Comment:
1. Please keep list in alphabetical order
2. Using your own version is great for testing, but when you know what you need and have tested it out, you need to open a PR for SGIS